docs/usage/upgrade-recommendations.md

```diff
@@ -0,0 +1,141 @@
+Again, start with the easy `major` version updates, like a Prettier or ESLint major update.
+
+Now it's time to work on the `major` update for your framework or library.
+Take your time, carefully read the changelogs, and make the necessary changes.
+Let multiple team members review your work before you merge it, it's easy to miss something.
+
+Finally, update your development tools.
+
+Now that you're up to date, it's important to start thinking about how to make updating a regular thing.
+
+## Project with five year old dependencies
+
+Let's assume your Dependency Dashboard lists more than 50 updates, and you have a few `major` version updates pending.
+If a project becomes this badly behind on updates, you have a bigger problem than just the updates.
+Let's deal with the easier stuff first: getting back up to date, and deal with the big problem after that.
+
+First update any dependencies that have critical updates for CVEs or other security related improvements.
+
+Next, update any dependency that's blocking another update.
+You may need to update dependency `A` before you can take an update for dependency `B` or `C`.
+In that case, update dependency `A` first.
+
+Then update all dependencies to their latest `minor` or `patch` version to prepare for dealing with `major` updates.
+
+We recommend you take `major` updates in sequence.
+Say you're on version `1` of dependency `A`, and the latest `major` version is at `4`.
+You should update to `2`, then `3` and finally `4`.
+Avoid updating from `1` directly to `4`.
+Taking `major` updates in sequence allows you to read the changelogs/blogs for each `major` version, and learn why upstream made certain breaking changes.
+
+Finally update your development tools.
+
+You're done with the _technical_ side of things, but you still need to work on the _human_ side.
+If you don't fix the human side, you'll end up with outdated dependencies again.
+Keep reading to learn how to deal with the human side of things.
+
+## Why developers avoid updating
+
+Let's assume that most developers _want_ a project that's up to date.
+So why are developers not updating their project?
+Some reasons why developers may avoid updating the project:
+
+- Developers get blamed when things break in production
+- There are no tests, so merging updates is scary
+- Slow tests
+- Releasing a new version of the project must be done by hand
+- Updating is a manual process
+- The company doesn't allow developer time for updates
+- The company has complex rules about when to update
+
+In short, if updating is painful, developers will avoid it.
+The solution is to make it easy and fast to update dependencies.
+Focus on the proces, not on the people.
+
+### Make updating easy and fast
+
+Insert recommendations from Renovate maintainers here on how to deal with a team that doesn't want to apply updates, arguments to convince people to update often, dealing with team dynamics, and so on.
```

Comment:
I would focus on the things that help convince people to update - ideally dealing with a team that doesn't update for whatever reason is simply showing them that it is beneficial for everyone by being a role model. This enables these teams to look at what you're doing and then adapt these things for their process. 

Showing them how easy it is will make them want these easy processes, too. At least, that's my experience.

Reply:
How easy updates are also depends on the quality and coverage of tests. To make updating as easy as it should be you need:

- Renovate bot
- Dependency Dashboard
- Right team mentality
- Company understands _why_ updating is generally safer than holding back update
- Tests for the critical path of the program

Updating _can_ be easy, but you have to work hard to get to the easy stuff. 😄